python/tvm/relay/op/_transform.py

```diff
@@ -705,6 +711,9 @@ def split_shape_func(attrs, inputs, _):
 
     axis = get_const_int(attrs.axis)
 
+    if axis < 0:
+        axis += inputs[0].shape[0]
```

Comment:
Don't you need to `get_const_int(inputs[0].shape[0])`?

Reply:
fixed